version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.8.0"
+	Version = "0.7.1"
```

Comment:
Should this change to something indicating TP1 as well?

Reply:
i'm inclined to say no, since it specifically represents the version of KnativeServing we're installing. I view TP1 as a bundle that includes KS.